### PR TITLE
As 59

### PR DESCRIPTION
**Pull request by SoundofGoodbye**
Task: https://github.com/SoundofGoodbye/movie-crawler/issues/59

### CODE REVIEW

File: apiservice/src/main/java/com/apiservice/services/movies/impl/TMDBMovieSearchServiceImpl.java [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
 
     @Override
     public String getMovie(String query, String language, int page, boolean includeAdults, String region, int year, int primaryReleaseDate) {
         queryBuilder = new SearchMovieQueryBuilder();
-        String tmdbMovieSearchQuery = queryBuilder.buildQuery(query, language, page, includeAdults, region, year, primaryReleaseDate);
+        String tmdbMovieSearchQuery = queryBuilder.buildQuery(includeAdults, region, year, primaryReleaseDate);
 
-        return apiService.query(getSearchType(), tmdbMovieSearchQuery);
+        return searchEngine.searchActors(SEARCH_OPTION, query + tmdbMovieSearchQuery, language, page);
     }
```
Comment (valjo94) [inline apiservice/src/main/java/com/apiservice/services/movies/impl/TMDBMovieSearchServiceImpl.java]: Not needed getMethod.

File: apiservice/src/main/java/com/apiservice/services/TMDBServiceImpl.java [hunk 2]
```diff
@@ -13,11 +13,12 @@
     @Value("${tmdb.api.key}")
     private String apiKey;
     private String url = "https://api.themoviedb.org/3/";
```
Comment (valjo94) [inline apiservice/src/main/java/com/apiservice/services/TMDBServiceImpl.java]: Make final and UPPER CASE the name.

**Review by valjo94 — CHANGES_REQUESTED** [review]

File: apiservice/src/main/java/com/apiservice/services/TMDBServiceImpl.java (hunk 2 above)
Comment (SoundofGoodbye) [reply apiservice/src/main/java/com/apiservice/services/TMDBServiceImpl.java]: done

File: apiservice/src/main/java/com/apiservice/services/movies/impl/TMDBMovieSearchServiceImpl.java (hunk 1 above)
Comment (SoundofGoodbye) [reply apiservice/src/main/java/com/apiservice/services/movies/impl/TMDBMovieSearchServiceImpl.java]: done

**Review by valjo94 — APPROVED** [review]

**Pull request merged**